src/service/video-manager-impl.js

```diff
@@ -221,11 +225,17 @@ class VideoEntry {
       // user interaction.
       if (this.video.element.hasAttribute(VideoAttributes.CONTROLS)) {
         this.video.hideControls();
+        this.autoplayIcon_ = this.createAutoplayIcon_();
+
+        this.vsync_.mutate(() => {
```

Comment:
I removed bunch of closures but I don't think vsync ones benefit from moving to a function since there is only one vsync closure per function scope in this case.